t/26-video_stream.t

```diff
@@ -291,6 +298,23 @@ subtest 'v4l2 resolution' => sub {
         [('/dev/video0', 'ssh host', '--set-dv-bt-timings query')],
         [('/dev/video0', 'ssh host', '--get-dv-timings')],
     ], "calls to v4l2-ctl";
+
+    @v4l2_ctl_calls = ();
+
+    # changed resolution
+    %v4l2_ctl_results = (
+        '--query-dv-timings' => '0',
+        '--get-dv-timings' => '1024x768p60',
+        '--set-dv-bt-timings query' => 'BT timings set',
+    );
+    $console->{dv_timings_last_check} = time - 4;
```

Comment:
I think calling `connect_remote` should covert this. `connect_remote` is called in other subtask. Can it also invoked here or is there any reason is not used?

Reply:
Hi @b10n1k 
I think `connect_remote` cannot cover this line: https://app.codecov.io/gh/os-autoinst/os-autoinst/blob/master/consoles%2Fvideo_stream.pm#L389
The above addition is covering this.

Reply:
Let me update the comment to make it a bit more clear

Reply:
Basically, the above line is just used to make the condition true written here https://github.com/os-autoinst/os-autoinst/blob/master/consoles/video_stream.pm#L380
Maybe we can also set `$console->{dv_timings_last_check}` to a known constant value such as:
`$console->{dv_timings_last_check} = $console->DV_TIMINGS_CHECK_INTERVAL;` to make the `if` condition true.